ospd/scan.py

```diff
@@ -378,6 +378,10 @@ def get_target_options(self, scan_id: str):
         """
         return self.scans_table[scan_id]['target'].get('options')
 
+    def get_target(self, scan_id: str) -> Dict:
```

Comment:
Would be nice to add the concrete Dict type here like `Dict[str, str]`. Not sure what key and value types are used here.

Reply:
Btw. same for the other methods like get_vts below